packages/frontend/src/pages/scaling-activity/view/ActivityPage.tsx

```diff
@@ -36,7 +36,7 @@ export function ActivityPage(props: ActivityPageProps) {
           tpsDaily={props.tpsDaily}
           tpsWeeklyChange={props.tpsWeeklyChange}
         />
-        <Chart endpoint={props.apiEndpoint} hideControls></Chart>
+        <Chart endpoint={props.apiEndpoint} hideControls={true}></Chart>
```

Comment:
is this in styleguide or what?

Reply:
It got there by the mistake, when I was resolving conflicts, changed it already. Thanks @krzkaczor